rascal-vscode-extension/src/lsp/RascalLSPConnection.ts

```diff
@@ -82,158 +85,6 @@ async function showContentPanel(url: string): Promise<void> {
         vscode.ViewColumn.One,
         {
             enableScripts: true,
```

Comment:
we don't need the portMapping anymore?

Reply:
In theory not; after reading the docs this should happen as a side-effect of the rewriting. So I've removed it tentatively.If we need it again later I'll open another PR and with a array splice instead of generated code.